src/ros.cpp

```diff
@@ -73,6 +73,32 @@ void RosPositionController::setTopic(QString topic)
 ///////////////////////////////////////////////////////////////////////////////
 ///////////////////////////////////////////////////////////////////////////////
 
+RosPosePublisher::RosPosePublisher(QQuickItem *parent):
+    _topic("pose")
+{
+	_publisher = _node.advertise<geometry_msgs::PoseStamped>(_topic.toStdString(), 1);
+    _seq = 0;
+}
+
+void RosPosePublisher::publish(QVector3D point, QQuaternion quaternion, QString frame){
```

Comment:
Why does the `publish` method needs a point + quaternion + frame? I believe the position + rotation should be the one of the `QQuickItem` (see [how I've done it for TF broadcaster](https://github.com/severin-lemaignan/ros-qml-plugin/blob/master/src/ros.cpp#L235) ) ; the frame should be accessed from the item `frame` property, as done with the other ROS-QML items.